firmware/integration/rusefi_config.txt

```diff
@@ -1556,6 +1557,9 @@ uint8_t[PEDAL_TO_TPS_SIZE] autoscale pedalToTpsRpmBins;;"RPM", 100, 0, 0, 25000,
 float[CLT_CRANKING_CURVE_SIZE] cltCrankingCorrBins;CLT-based cranking position multiplier for simple manual idle controller;"C", 1, 0, -100, 250, 2
 float[CLT_CRANKING_CURVE_SIZE] cltCrankingCorr    ;CLT-based cranking position multiplier for simple manual idle controller;"%", 1, 0, 0, 500, 2
 
+float[CLT_CRANKING_CURVE_SIZE] cltCrankingTaperCorrBins;CLT-based taper duration multiplier for simple manual idle controller;"C", 1, 0, -100, 250, 2
+float[CLT_CRANKING_CURVE_SIZE] cltCrankingTaperCorr    ;CLT-based taper duration multiplier for simple manual idle controller;"%", 1, 0, 0, 500, 2
```

Comment:
1. is 500 a reasonable maximum value for this parameter? maybe 5 is safer?
2. using float here is a waste of storage, maybe int8_t for temp, and uint8_t with scale 0.02 for multiplier would be better?

Reply:
Maybe a good idea to also make these change this for the `cltCrankingCorr`

Reply:
yes, finding other tables we can compact would be a great follow-on task #371